src/components/card.js

```diff
@@ -67,6 +67,22 @@ function Card() {
               title="Useful Tools"
             ></button>
           </div>
+          <div class="button-wrapper">
```

Comment:
Can you add a screenshot of the website after these changes are applied so that I can see how the UI looks now